### ISSUE TITLE
Batching rule for 'jax.nn.relu' not implemented

### ISSUE THREAD

**Issue by benmacadam64** [enpasos/jax2onnx#113]
Hi there,

This seems to generally be the case for whenever you vmap through an elementwise activation function (sigmoid also fails). I'm guessing the root issue is `lower_unary_elementwise`.
